netconf.c

```diff
@@ -268,6 +268,22 @@ _free_error_parms (nc_error_parms error_parms)
     g_hash_table_destroy (error_parms.info);
 }
 
+void
+apteryx_query_node_path (const GNode *node, GString* path)
+{
+    if (!node)
+        return;
+
+    const gchar* nname = (const gchar*) node->data;
+    g_string_append_printf(path, "/%s", nname);
+
+    GNode *child = node->children;
+    while (child != NULL)
```

Comment:
I'm a bit concerned about how this code works. OK if there is only ever one child, but what happens when there are more?